config-templates/redhat/odf-local/4.13/custom-parameters.json

```diff
@@ -0,0 +1,211 @@
+    "displayname": "IAM API key",
+    "name": "iam-api-key",
+    "category": "secret",
+    "obfuscate": "true",
+    "required": "true",
+    "mutable": "true"
+  },
+  {
+    "description": "Set to 'true' if you want to perform complete cleanup of ODF on assignment deletion.",
+    "displayname": "Perform Cleanup",
+    "name": "perform-cleanup",
+    "default": "false",
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "required": "true",
+    "type": "boolean",
+    "mutable": "true"
+  },
+  {
+    "description": "Set to 'true' if you want to enable storage class encryption.",
+    "displayname": "KMS encryption",
+    "name": "kms-encryption",
+    "default": "false",
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "required": "true",
+    "type": "boolean"
+  },
+  {
+    "description": "Your KMS instance name. The instance name must only include alphanumeric characters, '-', '_' or '.' and start and end with an alphanumeric character.",
+    "displayname": "KMS instance name",
+    "name": "kms-instance-name",
+    "regex": "^([a-zA-Z0-9]*[a-zA-Z0-9-_.]*[a-zA-Z0-9])$",
+    "min-length": 1,
+    "max-length": 63,
+    "required": "false"
+  },
+  {
+    "description": " Your KMS instance id.",
+    "displayname": "KMS instance id",
+    "name": "kms-instance-id",
+    "required": "false"
+  },
+  {
+    "description": " Your KMS instance public URL to connect to the instance.",
+    "displayname": "KMS instance Base URL",
+    "name": "kms-base-url",
+    "required": "false"
+  },
+  {
+    "description": "API key token URL to generate token for KMS instance.",
+    "displayname": "KMS instance API key token URL",
+    "name": "kms-token-url",
+    "place-holder": "https://iam.cloud.ibm.com/oidc/token",
+    "required": "false"
+  },
+  {
+    "description": "KMS root key of your instance.",
+    "displayname": "KMS root key",
+    "name": "kms-root-key",
+    "required": "false",
+    "category": "secret",
+    "obfuscate": "true"
+  },
+  {
+    "description": "IAM API key to access the KMS instance. The API key that you provide must have at least Viewer access to the KMS instance.",
+    "displayname": "KMS IAM API key",
+    "name": "kms-api-key",
+    "required": "false",
+    "category": "secret",
+    "obfuscate": "true"
+  },
+  {
+    "description": "Set to 'true' if you do not want to deploy MultiCloud Object Gateway (Noobaa)",
+    "displayname": "Ignore Noobaa",
+    "name": "ignore-noobaa",
+    "default": "false",
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "type": "boolean",
+    "required": "true"
+  },
+  {
+    "description": "Set to 'true' if you want to enable in-transit encryption",
+    "displayname": "In-transit Encryption",
+    "name": "encryption-intransit",
```

Comment:
I dont see this parameter getting used in `deployment.yaml`. Can you please check?